modules/openapi-generator/src/test/java/org/openapitools/codegen/options/BashClientOptionsProvider.java

```diff
@@ -40,6 +40,7 @@ public class BashClientOptionsProvider implements OptionsProvider {
                                 = "false";
     public static final String PREPEND_FORM_OR_BODY_PARAMETERS_VALUE = "true";
     public static final String ENUM_UNKNOWN_DEFAULT_CASE_VALUE = "false";
+    public static final String REMOVE_ENUM_VALUE_PREFIX_VALUE = "true";
```

Comment:
Why are you adding this here if you're referencing the default codegen key?

My understanding is that these are for generator specific keys (which you aren't using)

Reply:
It was breaking the tests saying that this new property needed to be added. I just followed the ENUM_UNKNOWN_DEFAULT_CASE_VALUE example to solve the test.

Reply:
Did this need to be defined here to just added to the builder below using `.put(CodegenConstants.REMOVE_ENUM_VALUE_PREFIX, REMOVE_ENUM_VALUE_PREFIX_VALUE)`?

This definition (`BashClientCodegen.REMOVE_ENUM_VALUE_PREFIX_VALUE`) isn't being used or referenced anywhere 

Which tests was failing?

Reply:
Exactly. I followed how others are doing in each test.
This fails:
![image](https://github.com/OpenAPITools/openapi-generator/assets/12387016/25e226cd-c92a-4266-b231-c8ff53655005)